docs/rules/no-underscore-dangle.md

```diff
@@ -42,6 +42,7 @@ This rule has an object option:
 * `"allow"` allows specified identifiers to have dangling underscores
 * `"allowAfterThis": false` (default) disallows dangling underscores in members of the `this` object
 * `"allowAfterSuper": false` (default) disallows dangling underscores in members of the `super` object
+* `"enforceInMethodNames": false (default) allows dangling underscores in method names`
```

Comment:
that would make sense, but I was confused about the other options.

`allowAfterSuper` => disallows?

I assumed the description would refer to the effect of the default option.